docs/md/flows.md

```diff
@@ -741,7 +741,14 @@ A [msg_created](sessions.html#event:msg_created) event will be created with the
 {
     "type": "send_msg",
     "uuid": "8eebd020-1af5-431c-b943-aa670fc74da9",
-    "text": "Hi @contact.name, are you ready to complete today's survey?"
+    "text": "Hi @contact.name, are you ready to complete today's survey?",
+    "template": {
+        "uuid": "3ce100b7-a734-4b4e-891b-350b1279ade2",
+        "name": "revive_issue",
```

Comment:
maybe this is where a `TemplateReference` could be used?

```
"templating": {
    "template": {
        "uuid": "3ce100b7-a734-4b4e-891b-350b1279ade2",
        "name": "revive_issue",
    },
    "variables": [
        "@contact.name"
    ]
}
```

Reply:
Sure if you prefer, let references spread far and wide!

Reply:
well it also helps with enumerating asset references in actions.. if this uses a template reference.. the action can just return that